helm/cluster-resources/values.schema.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
After generating this file, we add `required` fields manually. Since this app is created by us automatically, it is not a big deal right now. Just keep it in your mind :)